packages/gen-ai-hub/src/orchestration/orchestration-filter-utility.ts

```diff
@@ -0,0 +1,21 @@
+import { AzureContentSafety, FilteringConfig } from './client/api/index.js';
+
+/**
+ * Convenience function to create Azure filters.
+ * @param filter - Filtering configuration for Azure filter.
+ * @returns An object with the Azure filtering configuration.
+ */
+export function createAzureFilter(filter: AzureContentSafety): FilteringConfig {
```

Comment:
We could consider:

```suggestion
export function createAzureFilter(filter: AzureContentSafety = undefined): FilteringConfig {
```

because of [this default behavior](https://github.tools.sap/AI/llm-orchestration/issues/466#issuecomment-7346528).
So, effectively:
```ts
createAzureFilter();    // okay
createAzureFilter({});  // illegal, throws
```

Reply:
I like the way to enable default filters (`createAzureFilter();`). But, I couldn't use the default value of `undefined` for the filter parameter as `Type 'undefined' is not assignable to type 'AzureContentSafety'.`
I used the default value of `{ default: true }` instead.